framework/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Top level account owner now has admin privileges on the apps and adapters
 - Multiple `AbstractAccount`s now don't overlap
 - Top level account owner can now claim pending sub-accounts directly
+- `Clearable` helper type was added to the messages where clearing optional state could be useful
```

Comment:
Why can't we just use `Option`?

Reply:
Because `Option<Option<T>>` is kind of an anti-pattern imo. It doesn't give any meaningful type information.

`Some(None)` would clear the value vs `Some(Clear)` with `Clearable`

Reply:
How json looks for `Some(None)`?

Reply:
honestly no idea...

Reply:
https://github.com/serde-rs/serde/issues/1042

I don't think it's possible to serialize into a `Some(None)` value actually.